applications/asset_tracker_v2/boards/thingy91x_nrf9151_ns.conf

```diff
@@ -27,3 +27,53 @@ CONFIG_ADXL367_REFERENCED_ACTIVITY_DETECTION_MODE=y
 CONFIG_ADXL367_ACTIVITY_THRESHOLD=1000
 # App does not set activity time, set to 0 to disable
 CONFIG_ADXL367_ACTIVITY_TIME=0
+
+# configs for the Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
```

Comment:
Yes, that's intentional. I think it makes sense to have wifi enabled by default in the asset tracker.

Reply:
Having Wi-Fi enabled by default is good. But given all these WiFi related configurations are here, have you tried how to disable WiFi, or location module? I'm wondering if it gets complex requiring kind of reverting of these configs.
We should probably have non-WiFi config in sample.yaml to make sure it compiles if that's something we want to support.

Reply:
My question still remains. How do you disable Wi-Fi from the compilation and I'd like to see that configuration compiled in sample.yaml.

Reply:
`CONFIG_WIFI=n` and `CONFIG_WPA_SUPP=n` should do it.

Reply:
> `CONFIG_WIFI=n` and `CONFIG_WPA_SUPP=n` should do it.

That'll cause a build assert with location library given there are many location library configurations in this file.
`CONFIG_WPA_SUPP=n` is not needed as it's set in this file.

This is how I got it compiled although there are 6 warnings from Kconfigs that we are setting that doesn't exist because of the disabled configs:

- CONFIG_WIFI=n
- CONFIG_LOCATION_REQUEST_DEFAULT_METHOD_SECOND_CELLULAR=y
- CONFIG_LOCATION_REQUEST_DEFAULT_METHOD_THIRD_NONE=y

Disabling WiFi with those Kconfigs is definitely not intuitive so if this is our solution, it has to be documented or even an overlay for no_wifi configuration made.

All this being said, there is significant memory penalty in disabling WiFi just with those 3 Kconfigs because WiFi overlay enables features that are not needed when WiFi is not present. In addition, some stacks are bigger with WiFi.

Memory report without WiFi with above three Kconfigs
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      347104 B       800 KB     42.37%
             RAM:      157472 B     211736 B     74.37%
        IDT_LIST:          0 GB        32 KB      0.00%
```

Memory report without WiFi when thingy91x_nrf9151_ns.overlay doesn't exist:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      282416 B       800 KB     34.47%
             RAM:      134600 B     203416 B     66.17%
        IDT_LIST:          0 GB        32 KB      0.00%
```

Reply:
The team owning ATv2 has discussed and they are OK doing these changes and making non-WiFi configuration with Thingy:91 X fiarly complicated.